Capri.Web/appsettings.json

```diff
@@ -4,6 +4,12 @@
     "MaxNumOfMasterProposalsPerPromoter": 2,
     "MaxNumOfBachelorProposalsPerPromoter" : 4
   },
+  "SieveSettings": {
+    "CaseSensitive": false,
+    "DefaultPageSize": -1,
+    "MaxPageSize": -1,
+    "ThrowExceptions": false
```

Comment:
What's that settings for?

Reply:
Those parameters are for Sieve configuration. I can remove this, but later we won't have a possibility to customize sieve